### PR TITLE
Task3 ready to review

### PR DESCRIPTION
**Pull request by HazRem**

### CODE REVIEW

File: Homework/Task3_1/Queue.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    {
+        return list.Count == 0;
+    }
+
+    public override string ToString()
+    {
+        return string.Join(", ", list);
+    }
+}
+
```
Comment (IgorKheidorov) [inline Homework/Task3_1/Queue.cs]: Move to a separate file, one class-one file

File: Homework/Task3_1/Queue.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
+﻿public interface IQueue<T> where T : struct
```
Comment (IgorKheidorov) [inline Homework/Task3_1/Queue.cs]: Move to a separate file, one class (interface, enum, etc)-one file

**Review by IgorKheidorov — APPROVED** [review]
Cool,  move entities to separate files (one type one -file), and merge

File: Homework/Task3_1/Queue.cs (hunk 2 above)
Comment (HazRem) [reply Homework/Task3_1/Queue.cs]: Moved to IQueue.cs

**Pull request merged**